Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py

```diff
@@ -198,7 +230,8 @@ def fetch_incidents():
 
     filter_alerts_kwargs = get_incident_init_params()
     items = sixgill_alerts_client.get_actionable_alerts_bulk(limit=max_incidents_to_return, from_date=last_fetch_time,
-                                                             sort_order='asc', **filter_alerts_kwargs)
+                                                             sort_order='asc', **filter_alerts_kwargs,
+                                                             organization_id=demisto.params().get('org_id', None))
```

Comment:
I can't find such an argument `org_id` in the configuration parameters in the yml file
Please add it there as well

Reply:
Added. Please check